{"name": "test_Get_target_device_info", "status": "broken", "statusDetails": {"message": "OSError: [Errno 22] Invalid argument: '.\\res.json'", "trace": "self = <Testcase_Devices.test_04_Get_target_device_info.Test_04_Get_target_device_info object at 0x0000014B8850E580>\n\n    @allure.feature('【需求点】：获取固定SN设备的信息')\n    @allure.story('【正常获取】')\n    @allure.severity('normal')\n    def test_Get_target_device_info(self):\n        ip = ReadIP().Read()\n        urls = ReadURL().Read()\n        device_sn = ReadSN().Read()\n        url1 = urls['url04']['url']\n        url = ip + url1\n        url = url.replace('sn',device_sn)\n        auth = ReadAuthorization().Read()\n        headers = {\n            'Authorization': auth,\n        }\n        method = 'get'\n        # 发送请求\n>       res = RequestTools().send_requests(method=method,url=url,header=headers)\n\ntest_04_Get_target_device_info.py:27: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <common.request_utill.RequestTools object at 0x0000014B8850EBB0>\nmethod = 'get'\nurl = 'http://10.10.10.137:80/deviceService/v1/devices/0000306E77162578?withStatus=true'\ndata = None\nheader = {'Authorization': 'Basic dXZsaWdodDp1dmxpZ2h0LXNlcnZlci1nM2E0QGlqaw=='}\n\n    def send_requests(self,method,url,data=None,header=None):\n          if method == 'get' or method == 'GET':\n             res = self.get_method(url,data,header)\n          elif method == 'post' or method =='POST':\n             res = self.post_method(url,data,header)\n          elif method == 'patch' or method == 'PATCH':\n             res = self.patch_method(url, data, header)\n          elif method == 'delete' or method == 'DELETE':\n             res = self.delete_method(url,data,header)\n          elif method == 'put' or method == 'PUT':\n             res = self.put_method(url,data,header)\n          else:\n             res = \"请求方式有误！\"\n          # res = json.dumps(res)\n          # res = json.loads(str(res).replace(\"'\", \"\\\"\"))\n          # res = json.loads(res)\n    \n>         with open('.\\res.json','r') as fr:\nE         OSError: [Errno 22] Invalid argument: '.\\res.json'\n\n..\\common\\request_utill.py:73: OSError"}, "start": 1671585689808, "stop": 1671585689824, "uuid": "974702bb-7047-42b0-b4e4-d3247706a990", "historyId": "41c9e7470f0ec327a6f02ec3be0974e5", "testCaseId": "497176acc6aa09da0bda66145c2df40c", "fullName": "Testcase_Devices.test_04_Get_target_device_info.Test_04_Get_target_device_info#test_Get_target_device_info", "labels": [{"name": "feature", "value": "【需求点】：获取固定SN设备的信息"}, {"name": "story", "value": "【正常获取】"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "Testcase_Devices"}, {"name": "suite", "value": "test_04_Get_target_device_info"}, {"name": "subSuite", "value": "Test_04_Get_target_device_info"}, {"name": "host", "value": "SIE180286"}, {"name": "thread", "value": "11608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcase_Devices.test_04_Get_target_device_info"}]}